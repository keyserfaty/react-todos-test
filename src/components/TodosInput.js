import React from 'react'

const TodosInput = ({ inputText, onInputChange, onAddTodoClick }) => (
  <div id="todos-input">
    <input
      type="text"
      value={inputText}
      onChange={onInputChange}
    />
    <button
      onClick={() => onAddTodoClick(inputText)}
    >Add todo!</button>
  </div>
);

export default TodosInput
