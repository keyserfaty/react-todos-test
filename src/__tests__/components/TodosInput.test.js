import React from 'react'
import renderer from 'react-test-renderer'

import TodosInput from '../../components/TodosInput'

it('should render a todos input component', () => {
  const tree = renderer.create(
    <TodosInput />
  ).toJSON();

  expect(tree).toMatchSnapshot()
});
