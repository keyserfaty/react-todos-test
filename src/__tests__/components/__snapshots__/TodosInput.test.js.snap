exports[`test should render a todos input component 1`] = `
<div
  id="todos-input">
  <input
    onChange={undefined}
    type="text"
    value={undefined} />
  <button
    onClick={[Function]}>
    Add todo!
  </button>
</div>
`;
